Admin/CommentAdmin.php

```diff
@@ -150,6 +151,10 @@ public function postUpdate($object)
      */
     public function setCommentManager(ManagerInterface $commentManager)
     {
+        if (!$commentManager instanceof CommentManagerInterface) {
+            trigger_error('The '.__METHOD__.' method expected a CommentManagerInterface since version 2.4', E_USER_DEPRECATED);
```

Comment:
Please follow this message convention: https://github.com/symfony/symfony/blob/eca3e37225614ab49feb8cd1763ea1ebee76edd7/src/Symfony/Component/OptionsResolver/OptionsResolver.php#L479

And use FQN for classes.